cypress/integration/inventory/keyboard-shortcuts.spec.js

```diff
@@ -2,8 +2,28 @@ import TopMenu from '../../support/fragments/topMenu';
 import testTypes from '../../support/dictionary/testTypes';
 import InventoryMainButton from '../../support/fragments/inventory/inventoryMainButton';
 import permissions from '../../support/dictionary/permissions';
+import getRandomPostfix from '../../support/utils/stringTools';
+import InventoryInstanceEdit from '../../support/fragments/inventory/InventoryInstanceEdit';
 
 let userId = '';
+const precedingTitleValue = `Preceding title test value ${getRandomPostfix()}`;
+const isbnValue = `ISBN test value ${getRandomPostfix()}`;
+const issnValue = `ISSN test value ${getRandomPostfix()}`;
+const hotKeys = {
```

Comment:
can we keep it into fragment and reuse if needed?

Reply:
Done